templates/docs/examples/patterns/search-and-filter/chip-overflow.html

```diff
@@ -50,7 +50,7 @@
     <form class="p-search-and-filter__box" data-overflowing="false">
       <label class="u-off-screen" for="search">Search and filter</label>
       <input autocomplete="off" class="p-search-and-filter__input" id="search" name="search" placeholder="Search and filter" type="search" value="">
-      <button alt="search" class="u-off-screen" type="submit">Search</button>
+      <button alt="search" class="p-search-and-filter__search-button u-off-screen" type="submit">Search</button>
```

Comment:
Now that we have a specific class for this button would be nice for it to implement the `u-off-screen` functionality (maybe it can be included/extended?), so there is no need to have both.

Reply:
Fixed